visual-dotnet/SauceLabs.Visual/FullPageConfig.cs

```diff
@@ -6,14 +7,35 @@ namespace SauceLabs.Visual
     public class FullPageConfig
     {
         public int? DelayAfterScrollMs { get; set; }
+        public bool? DisableCSSAnimation { get; set; }
         public IEnumerable<string>? HideAfterFirstScroll { get; set; }
+        public bool? HideScrollBars { get; set; }
+        private int? _scrollLimit;
+        public int? ScrollLimit
+        {
+            get { return _scrollLimit; }
+            set
+            {
+                if (value > 0 && value <= 10)
+                {
+                    _scrollLimit = value;
+                }
+                else
+                {
+                    throw new ArgumentOutOfRangeException(nameof(ScrollLimit), "Value must be between 1 and 10.");
+                }
```

Comment:
Same as Java, the upper limit should be set only by the backend.

```suggestion
                if (value < 0)
                {
                    throw new ArgumentOutOfRangeException(nameof(ScrollLimit), "Value must be between 1 and 10.");
                }
                _scrollLimit = value;
```

or

```suggestion
                _scrollLimit = value;
                if (_scrollLimit < 0)
                {
                    _scrollLimit = 0;
                }
```